main.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"database/sql"
+	"go_final_project/database"
+	"go_final_project/handlers"
+	"log"
+	"net/http"
+	"os"
+	"path/filepath"
+)
+
+func main() {
+	dbFile := filepath.Join(".", "scheduler.db")
+	_, err := os.Stat(dbFile)
+
+	if os.IsNotExist(err) {
+		log.Println("Database file does not exist. Creating a new one.")
+		database.CreateDB(dbFile)
+	} else if err != nil {
+		log.Fatalf("Ошибка при проверке файла базы данных: %v", err)
+	} else {
+		log.Println("Database file exists.")
+	}
+
+	db, err := sql.Open("sqlite3", dbFile)
+	if err != nil {
+		log.Println(err)
+		return
+	}
+	defer db.Close()
+
+	port := os.Getenv("TODO_PORT")
+	if port == "" {
+		port = "7540"
+	}
+
+	mux := http.NewServeMux()
+	webDir := "./web"
+	mux.Handle("/", http.FileServer(http.Dir(webDir)))
+	mux.HandleFunc("/api/nextdate", handlers.NextDateHandler)
+	mux.HandleFunc("/api/task", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
Для красоты можно сделать тут так
```suggestion
	mux.HandleFunc("/api/task", handlers.TaskHandler(db))
```

Reply:
А функция будет выглядеть так
```golang
func TaskHandler(db *sql.DB) func(w http.ResponseWriter, r *http.Request) {
	return func(w http.ResponseWriter, r *http.Request) {
```